onnxruntime/core/mlas/lib/dwconv.cpp

```diff
@@ -72,7 +72,7 @@ MlasConvDepthwiseKernel(
         }
 
         if (c > 0) {
-            MLAS_FLOAT16X4 Accumulator = MlasZeroFloat16x4();
+            MLAS_FLOAT16X4 Accumulator = Bias == nullptr ? MlasZeroFloat16x4() : MlasLoadFloat16x4(&Bias[ChannelOffset]);
```

Comment:
is it safe to load 4 values from `&Bias[ChannelOffset]` in the `c < 4` case?

Reply:
Implement MlasLoadPartialFloat16x4